src/main/java/de/hpi/bpt/argos/core/ArgosImpl.java

```diff
@@ -48,7 +53,11 @@ public void start() {
 
 		Set<RestEndpoint> restEndpoints = new HashSet<>();
 		restEndpoints.add(eventReceiver);
-		// TODO: add more restEndpoints here
+		restEndpoints.add(new EntityEndpointImpl());
+		restEndpoints.add(new EntityMappingEndpointImpl());
+		//restEndpoints.add(new EntityTypeEndpointImpl());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)